ibis/expr/operations/structs.py

```diff
@@ -34,7 +34,9 @@ class StructColumn(Value):
 
     shape = rlz.shape_like("values")
 
-    def __init__(self, names, values):
+    def __init__(self, names: VarTuple[str], values: VarTuple[Value]):
+        if len(names) == 0:
+            raise ValidationError("StructColumn must have at least one field")
```

Comment:
Please remove this change. We haven't decided whether to continue to allow empty structs. For now, they are allowed.

```suggestion
```

Reply:
hmm, yeah IDK why I added this, thanks!

Reply:
Actually, empty structs are currently disallowed. But working on fixing that in https://github.com/ibis-project/ibis/pull/9310